src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball[] balls = new Ball[3];
+        for (Ball ball: balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
```suggestion
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
Good job, but we don't really need array here, we only need to print a ball three times
format your files before committing - Ctrl Alt L - shortcut to trigger IDEA's autoformatter, there should be space before and after : (semicolon)